types/react-navigation/react-navigation-tests.tsx

```diff
@@ -140,6 +144,7 @@ export const AppNavigator = createStackNavigator(
     {
         initialRouteName: ROUTE_NAME_START_SCREEN,
         initialRouteKey: ROUTE_KEY_START_SCREEN,
+        headerLayoutPreset,
```

Comment:
and this

Reply:
N (undo the shorthand and use `headerLayoutPreset: 'center',`)